docs/team/sbhbenjamin.md

```diff
@@ -11,16 +11,65 @@ Harmonia is a command-line-based personal task manager, which helps university s
+- Update User Preferences
+- Update`Model`
 
-* **Contributions to the UG**: To be added.
+**Priority feature:** [#153](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/153)
+- What it does: a feature that allows the user to attach priorities to tasks
+- Justification: This feature enhances the product as it allows a user to tag attach an importance to their task, to ensure that they can quickly prioritise what they want to do
+- Highlights: This enhancement affects existing commands such as `Add`, `Edit`, `Find` and `Sort` commands.
 
-* **Contributions to the DG**: To be added.
+**Conduct testing:**
+- Tests for the model [#147](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/147)
+- Tests for the `Task` and its properties (`Completion Status`, `Deadline`, `Description`, `Name`, `Task`, `NameContainsKeywordsPredicateTest`, `UniqueTaskListTest`)
+- Test for `TaskList` and `ModelManager`
+- Test for Tags [#271](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/271)
 
-* **Contributions to team-based tasks**: To be added.
 
-* **Review/mentoring contributions**: To be added.
+**Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=sbhbenjamin&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-02-18&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
 
-* **Contributions beyond the project team**: To be added.
+**Project management**
+- Managed releases [`v1.3`](https://github.com/AY2122S2-CS2103T-T09-1/tp/releases/tag/v1.3), [`v1.3b.trial`](https://github.com/AY2122S2-CS2103T-T09-1/tp/releases/tag/v1.3b.trial) (2 releases) on GitHub
+
+
+**Documentation**
+- User Guide
+    - Added documentation for the About section (Purpose, How to use the guide, Typography, Special Symbols) [#184](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/184)
+    - Added documentation to explain the GUI [#207](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/207)
+    - Added images of the GUI to better explain features [#278](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/278)
+    - Conducted cosmetic tweaks to improve consistency in terms of formatting and language used [#194](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/194)
+- Developer Guide
+    - Added documentation for the `Priority` feature [#158](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/158)
+    - Added documentation for the Model in the design component [#283](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/283)
+    - Added documentation for Harmonia's target user profile, value proposition, user stories, NFR and glossary [#57](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/57)
+    - Added documentation for use cases [#107](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/107)
+    - Conducted cosmetic tweaks to improve consistency in terms of formatting and language used [#107](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/107)
+
+
+**Team**
```

Comment:
not sure if it was meant to be like that but only this section was written in present tense, might want to consider changing it to past tense!!

Reply:
Just realised it too, thanks for spotting that! 👍